actioncable/lib/action_cable/connection/client_socket.rb

```diff
@@ -72,6 +73,7 @@ def rack_response
       def write(data)
         @stream.write(data)
       rescue => e
+        @logger.error "An #{e.class} error occurred while writing to the ActionCable socket: #{e.message}"
```

Comment:
Yeah I just thought of the same. I'll amend this. Currently also looking into why some tests fail.